src/Umbraco.Web/Runtime/WebRuntimeComponent.cs

```diff
@@ -127,8 +127,12 @@ public override void Compose(Composition composition)
             composition.Container.RegisterUmbracoControllers(typeLoader, GetType().Assembly);
             composition.Container.EnableWebApi(GlobalConfiguration.Configuration);
 
+            //we aren't scanning for ISearchableTree since that is not IDiscoverable, instead we'll just filter what we've
```

Comment:
Could we make `ISearchableTree` be `IDiscoverable`?

Reply:
@zpqrtbnk we could, i suppose it wouldn't make a performance difference would it?

Reply:
ah, 'cos `UmbracoApiController` is `IDiscoverable` so yes, wouldn't make a difference, perfs-wise, but would remove the quirks in `WebRuntimeComponent`. Thoughts?